server/routes/order.js

```diff
@@ -0,0 +1,61 @@
+const { Chef,Customer,Order, User } = require("../models/index");
+const express = require("express");
+const router = express.Router();
+const _ = require("lodash");
+const Joi = require("joi");
+const createError = require("http-errors");
+const { decodeToken, userIsAuthorized } = require("../middleware/auth");
+const fileUploadService = require("../services/fileUploader");
+
+/**
+ * GET chef's or customer's orders
+ */
+router.get("/:user_id", async (req, res, next) => {
+
+    const user = await User.findById(req.params.user_id);
+
+  res.status(200).send(user.orders);
```

Comment:
Does this send the actual order objects, or just their id's? I think the default behavior when you retrieve a document that has ObjectIds stored on it is that you need to call populate on the returned model in order to get the other documents referenced by the ids.

Reply:
Got it i will change that